book/docs/.ipynb_checkpoints/Open_data_code-checkpoint.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<p>Are you using 'modules' and 'functions' interchangeably? </p><p>We mention this is for python, perhaps link out or mention that such style guides exist in other languages.</p><p> </p><p>In addition to lower_case (this is actually called snake_case), I also adhere to the practice of nouns for variables and verbs+noun for functions</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/scaling_science/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
>Are you using 'modules' and 'functions' interchangeably?

I am not, seeing as functions can be a part of a module. Hope the adjustments are now clear to convey that.  

>We mention this is for python, perhaps link out or mention that such style guides exist in other languages.

Maybe you can provide guidance on style guides exist for other languages? 

>In addition to lower_case (this is actually called snake_case), I also adhere to the practice of nouns for variables and verbs+noun for functions

Adjusted to support this.